src/constants/tokens.ts

```diff
@@ -47,17 +47,17 @@ export const USDC_ARBITRUM_GOERLI = new Token(
 )
 export const USDC_POLYGON = new Token(
   ChainId.POLYGON,
-  '0x2791bca1f2de4661ed88a30c99a7a9449aa84174',
+  '0x3c499c542cef5e3811e1192ce70d8cc03d5c3359',
   6,
   'USDC',
   'USD//C'
```

Comment:
should this name be updated like with the mumbai one?

Reply:
I am not actually sure if the name here matters for anything - only the chainId, address, decimals, and symbol get used afai can tell. In theory though USDC's full name should be `USD Coin`